src/components/accordion/index.md.njk

```diff
@@ -17,83 +17,119 @@ The accordion component lets users show and hide sections of related content on
+
+Accordions can also work well for people who use a service regularly. For example, users of caseworking systems who need to do familiar tasks quickly.
+
+Test with users to decide if using an accordion outweighs the potential problems with hiding content.
 
 ## When not to use this component
 
-Accordions hide content from users and not everyone will notice them or understand how they work. For this reason do not use an accordion for content which is essential to all users.
+Accordions hide content from the user. Not all users will notice them or understand how they work. For this reason, you should only use them in specific situations and if user research supports it. 
+
+Do not use an accordion for content that all users need to see.
 
-Test your content without an accordion first. Consider if it’s better to:
+Test your content without an accordion first. Well-written and structured content, as shown in the [Content design: writing for GOV.UK](https://www.gov.uk/guidance/content-design/writing-for-gov-uk) guidance, can remove the need to use an accordion. 
+
+It’s usually better to:
 
 - simplify and reduce the amount of content
 - split the content across multiple pages
 - keep the content on a single page, separated by headings
-- use a list of links to let users navigate quickly to specific sections of content
-
-Do not use the accordion component if the amount of content it would need to contain will make the page slow to load.
+- use a list of links at the start of the page (known as 'anchor links') to take the user to particular sections of a page
 
-Accordions work best for simple content and links. Do not use accordions to split up a series of questions into sections. Use separate pages instead.
+Accordions work best for simple content and links. Do not use accordions to split up a series of questions. Use separate pages instead.
 
-## Accordions, tabs and details
+Avoid putting accordions within accordions, as it will make content difficult to find.
 
-Accordions, [tabs](/components/tabs/) and [details](/components/details/) all hide sections of content which a user can choose to reveal.
-
-Consider using an accordion instead of tabs if users might need to view more than one section at a time.
+Do not use the accordion component if the amount of content it would need to contain will make the page slow to load.
 
-You should also take into account the number of sections of content – accordions display vertically, so they can fit more sections than horizontal tabs.
+## Decide between using accordions, tabs and details
 
-Tabs may work better for users who need to switch quickly between 2 sections. Accordions push other sections down the page when they open, but tabs do not move which makes it easier to switch.
+Accordions, [tabs](/components/tabs/) and [details](/components/details/) all work by hiding sections of content which a user can choose to reveal. Be careful when using any of these components within one another.
```

Comment:
@christopherthomasdesign
Thanks for the comment re: nested accordions above. Any thoughts on this sentence down here about nesting a mix of these components?
Should I strengthen this instance to 'Do not use...' or maybe 'Avoid using...'?

```suggestion
Accordions, [tabs](/components/tabs/) and [details](/components/details/) all work by hiding sections of content which a user can choose to reveal. Be careful when using any of these components within one another.
```

Reply:
'Avoid' seems cautionary enough IMO – I think it suggests you should try other solutions first before nesting these components, and to test it well. Makes sense for us to advise that in cases where we're pretty sure it would be confusing but haven't tested it extensively ourselves.